src/batched-json.js

```diff
@@ -19,6 +19,23 @@ import {assertHttpsUrl} from './url';
 import {getValueForExpr} from './json';
 import {user} from './log';
 
+/**
+ * Detail of each `options` property:
+ * expr - Dot-syntax reference to subdata of JSON result to return. If not specified,
+ *     entire JSON result is returned.
+ * urlReplacement - If ALL, replaces all URL vars. If OPT_IN, replaces allowlisted
+ *     URL vars. Otherwise, don't expand.
+ * refresh - Forces refresh of browser cache.
+ * xssiPrefix - Prefix to optionally strip from the response before calling parseJson.
+ *
+ * @typedef {{
+ *  expr:(string|undefined),
+ *  urlReplacement: (UrlReplacementPolicy|undefined),
+ *  refresh: (boolean|undefined),
+ *  xssiPrefix: (string|undefined)}}
+ */
+export let FetchOptionsDef;
```

Comment:
naming suggestion ⬇️  
```suggestion
export let BatchFetchOptionsDef;
```

Reply:
I agree unless the object is the same as what a non-batched function would take (existing or not).

Reply:
Currently, its only used in batched function.